regression/cbmc-library/alloca_declaration/missing_parameters.c

```diff
@@ -0,0 +1,13 @@
+#include <assert.h>
+#include <stddef.h>
+
+void *alloca();
+
+int main()
+{
+  int n;
+  __CPROVER_assume(5 <= n && n < 10);
+  int *C = alloca();
```

Comment:
⛏️⛏️ capital C looks a bit unusual.

Reply:
It is indeed a little unusual. This was copy and pasted from this shadow memory test here - https://github.com/diffblue/cbmc/blame/2daecc3bf6c079f7a2eda826bf00e1f44186489d/regression/cbmc-shadow-memory/nondet-size-arrays1/main.c#L85 :grin: 

I'll fix the capitalisation in my new tests before I merge.